### PR TITLE
Adds Java interop test component.

### PR DESCRIPTION
**Pull request by dinooliva**

### CODE REVIEW

File: interoptest/src/javaservice/src/main/java/io/opencensus/interop/ServiceHopper.java [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+      long id, String name, String host, int port, TestRequest request) {
+    ManagedChannel channel = ManagedChannelBuilder.forAddress(host, port)
+                             // Channels are secure by default (via SSL/TLS).
+                             // For the test, we disable TLS to avoid
+                             // needing certificates.
+                             .usePlaintext(true)
+                             .build();
+    TestExecutionServiceGrpc.TestExecutionServiceBlockingStub blockingStub =
+        TestExecutionServiceGrpc.newBlockingStub(channel);
+    TestResponse response = blockingStub.test(request);
```
Comment (rghetia) [inline interoptest/src/javaservice/src/main/java/io/opencensus/interop/ServiceHopper.java]: would it block forever or would it timeout? If timeout occurs then the response should be a failure.

File: interoptest/src/javaservice/src/main/java/io/opencensus/interop/ServiceHopper.java [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+  }
+
+  private static TestResponse httpServiceHop(long id, String name, String host, int port,
+      TestRequest request, OcJettyHttpClient httpClient) {
+    try {
+      HttpRequest httpRequest = (HttpRequest) httpClient
+          .newRequest("http://" + host + ":" + port + "/test/request")
+          .method(HttpMethod.POST);
+      httpRequest.content(new StringContentProvider(request.toString()));
+      ContentResponse httpResponse = httpRequest.send();
```
Comment (rghetia) [inline interoptest/src/javaservice/src/main/java/io/opencensus/interop/ServiceHopper.java]: ditto.

File: interoptest/src/javaservice/src/main/java/io/opencensus/interop/ServiceHopper.java (hunk 1 above)
Comment (dinooliva) [reply interoptest/src/javaservice/src/main/java/io/opencensus/interop/ServiceHopper.java]: That's an interesting point - it's probably better to set a timeout and return a failure.  I've added a 1 second timeout here and below but we should be consistent across languages about a reasonable timeout period for these tests.

Consider adding that to the interoperability_test.proto.

File: interoptest/src/javaservice/src/main/java/io/opencensus/interop/ServiceHopper.java (hunk 2 above)
Comment (dinooliva) [reply interoptest/src/javaservice/src/main/java/io/opencensus/interop/ServiceHopper.java]: As per previous comment.

**Pull request merged**